docs/content/integration/openid-connect/kiali/index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Kiali"
+description: "Integrating Kiali with the Authelia OpenID Connect 1.0 Provider."
+summary: ""
+date: 2024-03-14T06:00:14+11:00
+draft: false
+images: []
+weight: 620
+toc: true
+support:
+  level: community
+  versions: true
+  integration: true
+seo:
+  title: "" # custom title (optional)
+  description: "Step-by-step guide to configuring Kiali with OpenID Connect 1.0 for secure SSO. Enhance your login flow using Authelia’s modern identity management."
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+## Tested Versions
+
+- [Authelia]
+  - [v4.39.5](https://github.com/authelia/authelia/releases/tag/v4.39.5)
```

Comment:
```suggestion
  - [v4.39.9](https://github.com/authelia/authelia/releases/tag/v4.39.5)
```

Suspect this has been tested with latest version?

Reply:
Yeah was an old PR, just missed it initially and incidentally before seeing your review haha